juju/series/series_windows_test.go

```diff
@@ -7,9 +7,11 @@ package series_test
 import (
 	"fmt"
 
+	"github.com/juju/testing"
+	"github.com/juju/utils"
```

Comment:
This won't compile under windows (duplicate import). Run sortimports on this, then (assuming you've got go 1.5), try

GOOS=windows go test -c

in this directory to check that it compiles.